### DISCUSSION TITLE
Weird extension limit

### DISCUSSION THREAD

**Discussion by NotFenixio** [TurboWarp/extensions#616] [Q&A] [answered]
When importing multiple extensions, i noticed that there's some kind of limit for the amount of extensions.
You see, i imported around 4 extensions, and then, when i tried to import a fifth one, it does nothing.

I checked for errors in the console, but nothing. Only the typical messages of "importing x extension".

Can someone tell me what's happening?

**Comment by CST1229** [comment]
I don't see any limit for the amount of extensions that can be added.
![](https://github.com/TurboWarp/extensions/assets/68464103/d54b06d0-9aa7-4d66-add0-a9e4aa490108)

**Reply by Flappy25** [reply]
I don't have any problems either

**Comment by GarboMuffin** [comment]
Which extensions are you importing?

Desktop app or website?

**Reply by NotFenixio** [reply]
Website.
I imported 3 (well, 4) extensions from the Extension Gallery: Time, Sheep_maker's Utilities, and Network.
Also, i imported an external extension, Meower Bot ([source](https://github.com/meower-community/Meowerbot-Scratch/blob/main/ext.js)). 
When I tried to import the Text extension, it just didn't do anything. It happens with all extensions i import from the gallery.

**Reply by NotFenixio** [reply]
Steps to reproduce:

1. Open a blank project on Turbowarp.
2. Open up the extension gallery.
3. Import Sheep_maker's Utilities from the blank project extension page.
4. Copy the URL of the extensions in the Extension Gallery called Time and Network, and import them.
You should have 3 extensions.
5. Go to [this page](https://github.com/meower-community/Meowerbot-Scratch/blob/main/ext.js) and copy the code.
6. Go to the Turbowarp project, and import the recently copied code unsandboxed. 
7. Try importing other extensions.

**Reply by GarboMuffin** [reply]
The third party extension you're trying to load is not a valid unsandboxed extension as it won't call Scratch.extensions.register, so we never get told that it finished loading.

**Reply by GarboMuffin** [reply, marked as answer]
It also lacks an IIFE to prevent interference with other extensions. So just load it with the sandbox or ask the creator to update it to support unsandboxed

**Reply by NotFenixio** [reply]
Thanks!
